source/rate-limit.ts

```diff
@@ -306,7 +306,7 @@ const handleAsyncErrors =
 			await Promise.resolve(fn(request, response, next)).catch(next)
 		} catch (error: unknown) {
 			/* istanbul ignore next */
-			next(error)
+			if (next) next(error)
```

Comment:
Why do we need the if here?

Reply:
The jest tests were crashing in between due to `next` not being a function, but that was while I was messing around with the `/crash` endpoint. Let me see if it works without that.